src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    static final String SECTION_DIVIDER = "____________________________________________________________";
+    private static void greetUser(){
+        String greeting = "\t" + SECTION_DIVIDER + "\n"
+                + "\tHello! I'm Duke. \n"
+                + "\tWhat can I do for you? \n"
+                + "\t" + SECTION_DIVIDER;
+        System.out.println(greeting);
+    }
+    private static void signOff(){
+        String sign_off = "\t" + SECTION_DIVIDER + "\n"
+                + "\tBye. Hope to see you again soon!\n"
+                + "\t" + SECTION_DIVIDER;
+        System.out.println(sign_off);
+    }
+    private static final String COMMAND_WORDS = "list" + "done" + "todo" + "deadline" + "event" + "delete";
+
+    static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void addTask(String taskDescription) throws
+            MissingDeadlineException, MissingEventTimeException {
+        Task t;
+        if (taskDescription.contains("todo")) {
+            if (taskDescription.length() < 5) {
+                throw new IndexOutOfBoundsException();
+            }
+            t = new Todo(taskDescription.substring(5));
+            System.out.println("\tGot it. I've added this task: ");
+        } else if (taskDescription.contains("deadline")) {
+            // Adds task as a Deadline
+            taskDescription.replace("deadline ", "");
+            int byIndex = taskDescription.indexOf("/by ");
+            if (byIndex == -1 || taskDescription.length() < byIndex + 4) {
```

Comment:
This expression could be difficult to qualify at first glance. See [https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions) for further suggestions.